pact/pact.py

```diff
@@ -71,6 +74,21 @@ def __init__(self, consumer, provider, host_name='localhost', port=1234,
         :param cors: Allow CORS OPTION requests to be accepted,
             defaults to False.
         :type cors: bool
+        :param publish_to_broker: Flag to control automatic publishing of
+            pacts to a pact broker. Defaults to False.
+        :type publish_to_broker: bool
+        :param broker_base_url: URL of the pact broker that pacts will be
+            published to. Defaults to None.
+        :type broker_base_url: str
+        :param broker_username: Username to use when connecting to the pact
+            broker if authentication is required. Defaults to None.
+        :type broker_username: str
+        :param broker_password: Password to use when connecting to the pact
+            broker if authentication is required. Defaults to None.
+        :type broker_password: str
+        :param broker_token: Authentication token to use when connecting to
```

Comment:
Would you mind putting in that this can also be supplied via an environment variable? We're trying to discourage people putting the token in the command line as it easily gets printed out. Actually, you can set PACT_BROKER_BASE_URL, PACT_BROKER_USERNAME, PACT_BROKER_PASSWORD and PACT_BROKER_TOKEN (I'm assuming that these env vars will be passed on to the child process automatically - in my experience, the generally are, but just wanted to check).

Reply:
Here is the proposed wording:
`broker_base_url` and `broker_username`: Can also be supplied through the PACT_BROKER_(BASE_URL/USERNAME) environment variable.
`broker_password` and `broker_token`: Strongly recommend supplying this value through the PACT_BROKER_(PASSWORD/TOKEN) environment variable instead.

Reply:
Sounds great to me.